apps/fluent-tester/src/TestComponents/Button/ButtonTest.tsx

```diff
@@ -64,10 +64,6 @@ const buttonSections: TestSection[] = [
       },
     ],
   }),
-  {
-    name: 'E2E Button Testing',
-    component: E2EButtonTest,
-  },
```

Comment:
Should this be removed?

Reply:
The section exists in the retractable E2E sections: https://github.com/microsoft/fluentui-react-native/blob/7464156fc6cef025f978f48a35b4dcb0dbcc017c/apps/fluent-tester/src/TestComponents/Button/ButtonTest.tsx#L69-L78

Reply:
I see, thanks!